ui/DesignSystem/Component/Copyable.svelte

```diff
@@ -23,6 +25,8 @@
       copyContent !== null ? copyContent : slotContent.textContent;
     copyToClipboard(content.trim());
 
+    notificationTxt ? notification.info(notificationTxt) : null;
```

Comment:
```suggestion
if(notificationTxt) {
  notification.info(notificationTxt);
}
```

The ternary operator feels a bit weird here.

Reply:
removed this and gave it a default text -> https://github.com/radicle-dev/radicle-upstream/pull/758/commits/0b9867696a1ef5ce4d154ef1d30462fa65af6006